samples/bluetooth/fast_pair/locator_tag/src/main.c

```diff
@@ -348,24 +376,21 @@ static void fmdn_provisioning_state_changed(bool provisioned)
 
 	/* Triggered on the unprovisioning operation. */
 	if (factory_reset_executed) {
+		LOG_INF("The device was unprovisioned and reset to factory settings");
 		LOG_INF("Please press a button to put the device in the Fast Pair discoverable "
 			"advertising mode after a reset to factory settings");
```

Comment:
The provisioning state change is already indicate in the log:

```suggestion
		LOG_INF("The device is reset to factory settings");
		LOG_INF("Please press a button to put the device in the Fast Pair discoverable "
			"advertising mode");
```

Reply:
you missed the the update of the second log